src/static/css/2019.css

```diff
@@ -898,7 +898,7 @@ p.copyright a {
 }
 
 .social-media a {
-  color: #8c99a3;
+  color: currentColor;
```

Comment:
Would removing this style entirely have the same effect?

Reply:
No, tried that as then it gets the link colour (purple) rather than the text colour (off white).

This is what ha-logo has too (presumably for the same reason):

https://github.com/HTTPArchive/almanac.httparchive.org/blob/2ea040c8dbea7d01cee640be13fe21c98e3a0011/src/static/css/2019.css#L828-L834